src/main/kotlin/com/chattriggers/ctjs/minecraft/libs/renderer/CTRenderPlayer.kt

```diff
@@ -100,11 +108,12 @@ internal class CTRenderPlayer(renderManager: RenderManager, useSmallArms: Boolea
         //$$     addLayer(CustomHeadLayer(this, context.modelSet))
         //$$     addLayer(ElytraLayer(this, context.modelSet))
         //$$ }
-        //$$
-        //$$ // TODO(CONVERT): Hide these behind flags?
-        //$$ addLayer(ParrotOnShoulderLayer(this, context.modelSet))
-        //$$ addLayer(SpinAttackEffectLayer(this, context.modelSet))
-        //$$ addLayer(BeeStingerLayer(this))
+        //$$ if (showParrot)
+        //$$     addLayer(ParrotOnShoulderLayer(this, context.modelSet))
+        //$$ if (showSpinAttackEffect)
+        //$$     addLayer(SpinAttackEffectLayer(this, context.modelSet))
+        //$$ if (showBeeStinger)
+        //$$     addLayer(BeeStingerLayer(this))
```

Comment:
These flags should be added to the method signature with default params

Reply:
Yeah I can do that.  There are a ton of layers other than those, should those be added as well?

Reply:
Also, should we have all of those layers as flags or just the ones that are useful to disable?

[Here's the list](https://nekoyue.github.io/ForgeJavaDocs-NG/javadoc/1.17.1/net/minecraft/client/renderer/entity/layers/package-summary.html) if you wanted to see them all, and it probably will keep increasing with more and more Minecraft versions.

Reply:
Yeah that's a bit much. Seems like something we can add later.